tests/foreman/cli/test_host.py

```diff
@@ -25,66 +37,497 @@ def setUp(self):
+        for name in valid_hosts_list():
+            with self.subTest(name):
+                host = entities.Host()
+                host.create_missing()
+                result = Host.create({
+                    u'architecture-id': host.architecture.id,
+                    u'domain-id': host.domain.id,
+                    u'environment-id': host.environment.id,
+                    # pylint:disable=no-member
+                    u'location-id': host.location.id,
+                    u'mac': host.mac,
+                    u'medium-id': host.medium.id,
+                    u'name': name,
+                    u'operatingsystem-id': host.operatingsystem.id,
+                    # pylint:disable=no-member
+                    u'organization-id': host.organization.id,
+                    u'partition-table-id': host.ptable.id,
+                    u'puppet-proxy-id': self.puppet_proxy['id'],
+                    u'root-pass': host.root_pass,
+                })
+                self.assertEqual(
+                    '{0}.{1}'.format(name, host.domain.read().name).lower(),
+                    result['name'],
+                )
+
+    @tier2
+    def test_positive_create_libvirt_without_mac(self):
+        """@Test: Create a libvirt host and not specify a MAC address.
 
+        @Feature: Hosts
+
+        @Assert: Host is created
         """
+        compute_resource = entities.LibvirtComputeResource(
+            url='qemu+tcp://{0}:16509/system'.format(
+                settings.server.hostname
+            ),
+        ).create()
         host = entities.Host()
         host.create_missing()
-        result = Host.create({
+        Host.create({
             u'architecture-id': host.architecture.id,
+            u'compute-resource-id': compute_resource.id,
             u'domain-id': host.domain.id,
             u'environment-id': host.environment.id,
+            u'interface': 'type=network',
             u'location-id': host.location.id,  # pylint:disable=no-member
-            u'mac': host.mac,
             u'medium-id': host.medium.id,
             u'name': host.name,
             u'operatingsystem-id': host.operatingsystem.id,
-            u'organization-id': host.organization.id,  # pylint:disable=E1101
+            # pylint:disable=no-member
+            u'organization-id': host.organization.id,
             u'partition-table-id': host.ptable.id,
             u'puppet-proxy-id': self.puppet_proxy['id'],
             u'root-pass': host.root_pass,
         })
-        self.assertEqual(
-            '{0}.{1}'.format(host.name, host.domain.read().name).lower(),
-            result['name'],
-        )
 
-    @tier2
-    def test_create_libvirt_without_mac(self):
-        """@Test: Create a libvirt host and not specify a MAC address.
+    @tier1
+    def test_positive_delete_by_id(self):
+        """@Test: Create a host and then delete it by id.
 
         @Feature: Hosts
 
-        @Assert: Host is created
+        @Assert: Host was deleted
```

Comment:
`Host is deleted` or otherwise use same language syntax above and below that code